shared/js/initApp.js

```diff
@@ -13,7 +13,8 @@ window.apps = {
   load: loadApp,
   // Legacy Blockly initialization that was moved here from _blockly.html.haml.
   // Modifies `appOptions` with some default values in `baseOptions`.
-  setup: function () {
+  // Move blockly-specific setup function out of shared and back into dashboard.
+  setupBlockly: function () {
```

Comment:
I changed when we were starting to move towards an app that could be run from a USB drive.  The idea was that a single html page could load many different types of apps based only on the values in `appOptions`.